packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -90,8 +91,13 @@ export class KonnectorModal extends PureComponent {
     })
   }
 
-  handleKonnectorJobError(trigger) {
-    this.setState({ isJobRunning: false, trigger })
+  async handleKonnectorJobError() {
+    const { fetchTrigger } = this.props
+    const { trigger } = this.state
+    this.setState({
+      isJobRunning: false,
+      trigger: await fetchTrigger(trigger._id)
```

Comment:
If the fetchTrigger does not succeed, the `isJobRunning` state attribute is not correctly set. I think the update of `trigger` and setting `isJobRunning` should be decoupled.

Reply:
Lazy me :)

Reply:
BTW I am not happy with this pattern. I guess something is wrong in the conception.